src/test/java/com/cardinity/CardinityIntegrationTest.java

```diff
@@ -121,6 +122,30 @@ public void testCreateDeclinedPayment() {
         assertFalse(resultPayment.isThreedsV1());
     }
 
+    @Test
+    public void testCreateApprovedPaymentWithDescriptorSuffix() {
+        Payment payment = getBaseCCPayment();
+        payment.setStatementDescriptorSuffix("Suffix");
```

Comment:
Lets use shorter suffix to reduce the risk of test failing due to too long statement descriptor suffix. Lets use suffix of 2 chars

Reply:
resolved 
https://github.com/cardinity/cardinity-sdk-java/pull/3/commits/5ff6e03cdbe6b5676919a77ff9410a38078739ff